product/outline.md

```diff
@@ -0,0 +1,44 @@
+# Audience
+
+While the content of this segment can be intersting to engineers in order to help their teams better navigate business decisions related to InnerSource the core audience are people in your team and across your organisation that deal with product development. The goal of this segment is to help you understand how InnerSource can help interdependent teams resolve requirements without syncing work to the point where both teams operate in lock-step.
+
+# Takeaways
+
+After going through this segment, you will:
+
+* understand how Agile software development relates to InnerSource:
+   * Which problems does InnerSource address that cause issues in typical agile teams? (Think "wait it out", "build workarounds" vs. "help with the solution".)
+   * In what aspects does InnerSource differ from agile or traditional software development - in particular when the same terminology and technology is used? (Think "Issue tracker for backlog/ board management vs. issue tracker for communication", "code review as quality gate vs. continuous code review as a conversation").
+*  understand how even with InnerSource the host team still has to plan time and capacity when receiving features as pull requests as opposed to features requests.
+*  understand additional negotiation possibilities that InnerSource brings (think "host teams implements tricky feature request of contributing team, in return the contributing team helps with simpler features that the host teams has higher up on their priority list")
+
+# Outline
+
+InnerSource and Agile
+
+- Issues InnerSource solves
+- Common mis-understandings
```

Comment:
That could certainly be part of it. Another thing I see: Agile uses certain tools and phrases. Sometimes InnerSource uses the same tools and phrases - but with different meaning or purpose.

If with an InnerSource background you talk to an Agile coach about issue trackers you may notice that they are using them as a digital board of sticky notes to track progress. In Open Source at least I see issue trackers as a place where people communicate, collect information, design, test and talk about architecture. As a result there are people from the agile community who see it as best practice to keep information in issues that are not currently under development as low as possible - this makes it easier to avoid the sunk cost fallacy when having to decide to archive issues that are not currently of interest - and sometimes never will be of interest.

Another example would be code reviews: In traditional software engineering those are done as well - but for the purpose of quality control, often after the entire code base has been written, commonly by an entity that is not related to the original team that did the implementation. In InnerSource we do pull requests which come with code review as well - but they are much more incremental, they can happen on work in progress pieces and they are much more a conversation instead of quality checking.

I'm sure together we can think of other examples where people coming from an agile background will be bound to understand what we talk about in the wrong way. The goal of this section would be to highlight those things so people stop talking past each other...

Reply:
Good points.  Putting some of those sub-bullets in the outline could be useful.

Reply:
I agree to @MaineC's points and we should add them.
There's a huge variation of what people claim to be agile, I've even seen partial cases where some agile practitioners pretty much were aligned with the innersource/opensource interpretation. So probably add a "YMMV, vastly"? :wink: 

Also, depending on the state of the company when starting any innersource efforts and the level of micropolitical activity in the company, you'll surface **very** strong effects as you highlight and impact all of these games being played. So maybe a  warning / notice might be warranted?